README.md

```diff
@@ -1,56 +1,93 @@
 ```bash
 git clone git@github.com:Rignchen/Depot.git
 cd Depot
 cargo build --release
 mv target/release/depot /usr/local/bin
 ```
+Please note that you need to have [Rust](https://www.rust-lang.org/tools/install) installed to build the project.
 
 ## Configuration
-Depot will try to guess your package manager depending on your operating system, however sometimes you may want to specify it yourself, for example on Arch Linux you may want to use `yay` instead of `pacman`.\
+Depot will try to guess your package manager based on your operating system. However, sometimes you may want to specify it yourself.
+For example, as I use Arch (btw), I may want to use `yay` instead of `pacman`.\
 To do so you have 2 options:
 1. Set the environment variable `DEPOT_PM` to the name of your package manager
 2. Add the flag `--pm <package_manager>` when running a command
 
 ## Usage
 Depot has 5 commands:
-- `install <package>`: Install a package
-- `uninstall <package>`: Uninstall a package
-- `update` <package>: Update a package, if no package is specified, update all the packages
-- `list`: List all the packages you have installed
-- `sync`: Compare the packages installed on the computer with the ones stored in the file and install the missing ones
+- `install <package>`: Install a package.
+- `uninstall <package>`: Uninstall a package.
+- `update [package]`: Update a package, if no package is specified, update all the packages.
+- `list`: List all the packages you have installed.
+- `sync`: Compare the packages installed on the computer with the ones stored in the file and install the missing ones.
+
 
 ## Example
-In this example I will install `neovim` and `zsh` on both my computers
+In this example I will install `vim` and `zsh` on both of my computers.\
+I use `scp` to copy the package list file from one computer to the other.\
+Then I use `ssh` to connect to the other computer and run `depot sync` to install the missing packages on the other computer.
 ```bash
-Depot install neovim
-Depot install zsh
-scp ~/.depot/packages user@computer:~/.depot/packages
-ssh user@computer
-Depot sync
+$ depot install vim
+vim is now installed.
+
+$ depot install zsh
+zsh is now installed.
+
+$ scp ~/.depot/packages user@computer:~/.depot/packages
+wanted.json                                     100% 123     2.2MB/s   00:00
+
+$ ssh user@computer 
+$ whoami
+user
+
+$ depot sync
+Packages synced with config.
+
+$ which vim
+/usr/bin/vim
 ```
 
-## Supported package managers
-- pacman
-- yay
-- apt
-- apt-get
-- pkg
-- dnf
-- apk
+## Supported package manager
+
+Package managers listed below are supported by Depot:
+
+| OS/Linux Distribution | `pacman` | `yay` | `apt` | `apt-get` | `pkg` | `dnf` | `apk` |
+|-----------------------|----------|-------|-------|-----------|-------|-------|-------|
+| Arch (btw)            | ✔        | ✔     | X     | X         | X     | X     | X     |
+| Debian/Ubuntu         | X        | X     | ✔     | ✔         | X     | X     | X     |
```

Comment:
Would be bether if Ubuntu and Debian were 2 distrinct lines as they are not the same operating system

Reply:
As Ubuntu is based on Debian, I put them together. But a separated line would be better, you're right

Reply:
maybe one day we'll add support for a package manager that only works on Ubuntu so it's better to have them as 2 lines